src/hooks/useAccountRiskCheck.ts

```diff
@@ -0,0 +1,42 @@
+import { sendEvent } from 'components/analytics'
+import ms from 'ms.macro'
+import { useEffect } from 'react'
+import { ApplicationModal, setOpenModal } from 'state/application/reducer'
+import { useAppDispatch } from 'state/hooks'
+
+export default function useAccountRiskCheck(account: string | null | undefined) {
+  const dispatch = useAppDispatch()
+
+  useEffect(() => {
+    if (account) {
+      const riskCheckLocalStorageKey = `risk-check-${account}`
+      const now = Date.now()
+      try {
+        const storedTime = localStorage.getItem(riskCheckLocalStorageKey)
+        const checkExpirationTime = storedTime ? parseInt(storedTime) : now - 1
+        if (checkExpirationTime < Date.now()) {
+          const headers = new Headers({ 'Content-Type': 'application/json' })
+          fetch('https://screening-worker.uniswap.workers.dev', {
```

Comment:
What is this?

Reply:
A new hook Uni added for checking of blocked accounts.
We are not using this for now